### DISCUSSION TITLE
I want to stop custom domains from being applied to new repositories

### DISCUSSION THREAD

**Discussion by 7name** [community/community#23524] [Other Feature Feedback, Questions, & Ideas]
<p>I don’t understand English, so please excuse the DeepL translation.</p>
<p>I have one GitHubPages with a custom domain and<br>
I want to create a new repository and a completely unrelated site.</p>
<p>But the custom domain is applied even in the new repository.<br>
→/.github.io</p>
<p>.github.io<br>
How can I do this?</p>
<p>Sorry if this is confusing.<br>
Thank you in advance.</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>Is your custom domain configured in the repository for your <em>user site</em> (<code>7name.github.io</code>)? If yes, it’ll be used for all your repository sites as well, unless you give them another custom domain.</p>
<p>If that’s the case, renaming the user site repository so it’s no longer your user site should help.</p>

**Comment by jfullstackdev** [comment]
<p>it seems this repo should not be under that custom domain,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/a/4/a40dcebce9a80a0f08444deb2f036127fe9d2e0c.png" data-download-href="/uploads/short-url/nphT2smZ6EMtHHMcZlODfLk6o0Y.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077547-e69709af-b788-4006-9754-e33d1fc524ec.png" alt="image" data-base62-sha1="nphT2smZ6EMtHHMcZlODfLk6o0Y" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/a/4/a40dcebce9a80a0f08444deb2f036127fe9d2e0c_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/a/4/a40dcebce9a80a0f08444deb2f036127fe9d2e0c_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/a/4/a40dcebce9a80a0f08444deb2f036127fe9d2e0c_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/a/4/a40dcebce9a80a0f08444deb2f036127fe9d2e0c_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 84.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>if that is the case, create an Org and it will be a separate one from your main account</p>

**Comment by 7name** [comment]
<p>(DeepL)<br>
It looks like the only way to do this is to use another custom domain or create it under a different account.</p>
<p>I thought I had missed something because there was not enough information.<br>
I was hoping that the custom domain would not be applied to new repositories, but if it is the spec, I have no choice.</p>
<p>Thank you for your help. Thank you very much.</p>

**Comment by jfullstackdev** [comment]
<aside class="quote no-group" data-username="7name" data-post="4" data-topic="248360">
<div class="title">
<div class="quote-controls"></div>
 7name:</div>
<blockquote>
<p>create it under a different account.</p>
</blockquote>
</aside>
<p>you don’t need to do that, simply create another org, it follows the same rule, when you enable GitHub Pages, you have a separate domain for that and you can even link that to your custom domain,</p>
<p><code>[orgname]</code>.github.io</p>
<p>as if this is acting like a separate account but it’s not, it’s simply your org, even if you don’t have members,</p>
<p>just like my sample org</p>
<p><a href="https://github.com/devz-throwable">https://github.com/devz-throwable</a></p>
<p>and it’s GitHub Page</p>
<p><a href="https://devz-throwable.github.io/">https://devz-throwable.github.io/</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/4/1/41c1e6cc34f96b4c2a97a4b4063280e8e63b7e5d.png" data-download-href="/uploads/short-url/9nIsZZTCgzZN0aSzIExjQsh2FYN.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077551-c3bacaba-70cc-44a7-a178-7e1a19613b44.png" alt="image" data-base62-sha1="9nIsZZTCgzZN0aSzIExjQsh2FYN" width="690" height="236" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/4/1/41c1e6cc34f96b4c2a97a4b4063280e8e63b7e5d_2_690x236.png, https://aws1.discourse-cdn.com/github/optimized/3X/4/1/41c1e6cc34f96b4c2a97a4b4063280e8e63b7e5d_2_1035x354.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/4/1/41c1e6cc34f96b4c2a97a4b4063280e8e63b7e5d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/4/1/41c1e6cc34f96b4c2a97a4b4063280e8e63b7e5d_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1095×376 14.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>see, there is not even <code>jdevstatic</code> at the URL</p>
